packages/neon-core/src/wallet/Account.ts

```diff
@@ -158,10 +158,7 @@ export class Account implements NeonObject<AccountJSON> {
   }
 
   public get isMultiSig(): boolean {
-    return (
-      this.contract?.script?.slice(this.contract.script.length - 8) ===
-      InteropServiceCode.NEO_CRYPTO_CHECKMULTISIGWITHECDSASECP256R1
-    );
+    return isMultiSigContract(this.contract.script);

```

Comment:
the contract script is stored in base64 format which is currently not taken into account. That means `isMultiSig` wil always return false

https://github.com/CityOfZion/neon-js/blob/c6dc38a180e1651d0abe7e2afce0107a5eea023f/packages/neon-core/src/wallet/Account.ts#L30-L32

Reply:
I am using VSCode oddly enough. Not sure what happened with line formatting. I can revisit that. 

As for the script, I messaged some thoughts on slack.